tests/init.js

```diff
@@ -24,11 +24,19 @@
 
 // Define global params needed for tests to run on existing app code
 var params = {};
+var webpMachine = true;
```

Comment:
I agree. I'll have a go.

Reply:
Given the refactoring happening in #672, I think it would be best to add this test after the refactor has been accepted, so I'll add a new issue for this.